tests/app/testappd/cmd/root.go

```diff
@@ -279,7 +279,7 @@ func txCommand() *cobra.Command {
 		authcmd.GetBroadcastCommand(),
 		authcmd.GetEncodeCommand(),
 		authcmd.GetDecodeCommand(),
-		authcmd.GetAuxToFeeCommand(),
+		// authcmd.GetAuxToFeeCommand(),
```

Comment:
nit: prefer removing, instead of commenting refer here: https://github.com/cosmos/cosmos-sdk/blob/b7749afcaf72702589d98fd00126ea26e2947ae5/simapp/simd/cmd/commands.go#L102

Reply:
https://github.com/skip-mev/block-sdk/pull/184/commits/0fadd8cec318279d11a32ffe199da59d08c59885